src/controllers/connectionManager.ts

```diff
@@ -1086,52 +1083,57 @@ export default class ConnectionManager {
 	public async refreshAzureAccountToken(uri: string): Promise<void> {
 		const profile = this.getConnectionInfo(uri);
 		if (!profile) {
-			this.vscodeWrapper.logToOutputChannel(Utils.formatString(LocalizedConstants.msgConnectionNotFound, uri));
+			this.vscodeWrapper.logToOutputChannel(LocalizedConstants.msgConnectionNotFound(uri));
 			return;
 		}
 
 		// Wait for the pending reconnction promise if any
 		const previousReconnectPromise = this._uriToConnectionPromiseMap.get(uri);
 		if (previousReconnectPromise) {
-			this.vscodeWrapper.logToOutputChannel(Utils.formatString(LocalizedConstants.msgFoundPendingReconnect, uri));
+			this.vscodeWrapper.logToOutputChannel(LocalizedConstants.msgFoundPendingReconnect(uri));
 			try {
 				const previousConnectionResult = await previousReconnectPromise;
 				if (previousConnectionResult) {
-					this.vscodeWrapper.logToOutputChannel(Utils.formatString(LocalizedConstants.msgPendingReconnectSuccess, uri));
+					this.vscodeWrapper.logToOutputChannel(LocalizedConstants.msgPendingReconnectSuccess(uri));
 					return;
 				}
-				this.vscodeWrapper.logToOutputChannel(Utils.formatString(LocalizedConstants.msgFoundPendingReconnectFailed, uri));
+				this.vscodeWrapper.logToOutputChannel(LocalizedConstants.msgFoundPendingReconnectFailed(uri));
 			} catch (err) {
-				this.vscodeWrapper.logToOutputChannel(Utils.formatString(LocalizedConstants.msgFoundPendingReconnectError, uri, err));
+				this.vscodeWrapper.logToOutputChannel(LocalizedConstants.msgFoundPendingReconnectError(uri, err));
 			}
 		}
 
 		const expiry = profile.credentials.expiresOn;
 		if (typeof expiry === 'number' && !Number.isNaN(expiry)) {
 			if (AzureController.isTokenExpired(expiry)) {
-				this.vscodeWrapper.logToOutputChannel(Utils.formatString(LocalizedConstants.msgAcessTokenExpired, profile.connectionId, uri));
+				this.vscodeWrapper.logToOutputChannel(LocalizedConstants.msgAcessTokenExpired(profile.connectionId, uri));
 				try {
 					let connectionResult = await this.connect(uri, profile.credentials);
 					if (!connectionResult) {
-						this.vscodeWrapper.showErrorMessage(Utils.formatString(LocalizedConstants.msgRefreshConnection, profile.connectionId, uri));
+						this.vscodeWrapper.showErrorMessage(LocalizedConstants.msgRefreshConnection(profile.connectionId, uri));
 						throw new Error('Unable to refresh connection');
 					}
-					this.vscodeWrapper.logToOutputChannel(Utils.formatString(LocalizedConstants.msgRefreshTokenSuccess,
-						profile.connectionId, uri, this.getConnectionInfo(uri)));
+					this.vscodeWrapper.logToOutputChannel(
+						LocalizedConstants.msgRefreshTokenSuccess(
+							profile.connectionId,
+							uri,
+							JSON.stringify(this.getConnectionInfo(uri))
+						)
+					);
 					return;
 				} catch {
 					this.vscodeWrapper.showInformationMessage(Utils.formatString(LocalizedConstants.msgRefreshTokenError));
```

Comment:
Is there a reason Utils.formatString is still needed here?

Reply:
Given that we have string interpolation for non-localized strings and function-based string construction for localized strings, think we could just get rid of `Utils.formatString` entirely?  Feels like a bad-practice function to keep around to me since it doesn't enforce or inform the number or order of string parameters.

Reply:
I don't know why this was added in the first place as there is nothing to format for the string.